### DISCUSSION TITLE
Where is the bank balance data stored for a town or nation?

### DISCUSSION THREAD

**Discussion by Naini0712** [TownyAdvanced/Towny#5054] [Q&A]
I have to check the bank balances of every town or nation to keep track of the server's finances.

**Comment by Semisol** [comment]
Let me compile a list.
`towny-war-chest`: War chest, sum of money left in the banks of towns and nations, this is not reliable as it gets reset/modified when a war happens.
`town-<name>` and `nation-<name>`: What they sound like

They are literal usernames so you type /bal name

**Reply by Semisol** [reply]
Done.

**Comment by LlmDl** [comment]
The actual balances are stored in your economy plugin. What @Semisol has said is accurate for most servers which haven't changed the economy prefixes (the `nation-` and `town-` parts in their reply.)
